src/modules/Packages.rb

```diff
@@ -2795,6 +2797,14 @@ def kept_products(products)
     def has_window_manager?
       Pkg.IsSelected("windowmanager") || Pkg.IsProvided("windowmanager")
     end
+
+    # Checking if two products are conflicting with each other.
```

Comment:
Um, it's not exactly `with each other`. It checks whether `product1` conflicts with `product2`. It is irrelevant whether `product2` conflicts with `product1` or not.

The check is only one way.